### ISSUE TITLE
#Feature

### ISSUE THREAD

**Issue by scheng123** [sharpie7/circuitjs1#169]
#Feature: Can you stablize the pwn signal when adjusting duty cycle? Something like this: 
![image](https://user-images.githubusercontent.com/16139772/30599554-dd037f08-9d2a-11e7-838a-65a9e989b5f3.png)

**Comment by scheng123** [comment]
Sorry, github didn't let me upload gifs, here's the original source: https://www.youtube.com/watch?v=YmPziPfaByw ; First 4 seconds of the video.

**Comment by scheng123** [comment]
Can you not make the signal move to the left? It always confuses me when I am trying to compare the duty cycle.

**Comment by sharpie7** [comment]
@scheng123 for the last point you can just hit the "RUN/Stop" button to freeze the scopes if you want to take measurements.

I have updated the title to reflect the PWM point.

**Comment by scheng123** [comment]
Yes, but I don't want to keep stopping and starting, and using a ruler, for every cycle, can you just "lively" take measurements for the duty cycle? Without the scopes moving? And doesn't hitting the "Run/Stop" stop the scope entirely, I don't want to continuously hit the Run/Stop button!

**Comment by scheng123** [comment]
Yes, but I don't want to keep stopping and starting, and using a ruler, for every cycle, can you just "lively" take measurements for the duty cycle? Without the scopes moving? And doesn't hitting the "Run/Stop" stop the scope entirely, I don't want to continuously hit the Run/Stop button!

**Comment by pfalstad** [comment]
you can do this with scope triggers now.

https://www.falstad.com/s.php?s=eCzjVc
